backend/internal/models/habit.go

```diff
@@ -1,7 +1,12 @@
 ﻿package models
 
 type Habit struct {
-	ID          int    `gorm:"primaryKey" json:"habit_id"`
-	HabitName   string `json:"habit_name"`
-	Description string `json:"description"`
+	ID            int      `gorm:"primaryKey" json:"habit_id"`
+	HabitName     string   `json:"habit_name"`
+	Description   *string  `json:"description"`
+	UserId        int      `json:"user_id"`
+	StartDate     string   `json:"start_date"`
+	Notifications *bool    `json:"notifications"`
+	TodayValue    *float32 `json:"today_value"`
+	Plans         []Plan   `gorm:"foreignKey:HabitId"`
```

Comment:
type PlanUnit string

// Defines values for PlanUnit.
const (
	Count    PlanUnit = "count"
	Distance PlanUnit = "distance"
	Time     PlanUnit = "time"
)